contributivity_measures.py

```diff
@@ -77,9 +94,10 @@ def compute_SV(node_list, epoch_count, x_val_global, y_val_global, x_test, y_tes
         # print('\nComputing characteristic function on coalition ', coalition) # VERBOSE
         characteristic_function.append(
             fl_training.compute_test_score(
-                coalition_nodes, epoch_count, x_val_global, y_val_global, x_test, y_test
+                coalition_nodes, epoch_count, x_val_global, y_val_global, x_test, y_test, aggregation_weighting, minibatch_count
             )
         )
+        the_scenario.fit_count += 1
```

Comment:
Ah OK, here too. So maybe this in an ambiguity in the definition or naming of `fit_count`. What quantity exactly are you trying to capture @Thomas-Galtier?

Reply:
I trying to track the number of times we compute the characteristic function.

Reply:
OK. So I think for clarity you should rename it, otherwise from time to time we'll be confused why `fit_count` doesn't count the number of `.fit` :-)
What you want to track is closer to "the number of model training and evaluation" isn't it?